### ISSUE TITLE
Big BUG on Windows with  uranium:UraniumContentPage.Attachments, material:BottomSheetView

### ISSUE THREAD

**Issue by GuyFlorent** [enisn/UraniumUI#721]
There is a serious bug on Windows related to the uranium
.Attachments component and the material : BottomSheetView
![Bug Uranium](https://github.com/user-attachments/assets/c709f9af-edad-4622-bdbc-66ce2bec8ec7)

. This component takes up the entire screen, preventing clear visibility of the content and blocking any interaction. You will find a screenshot attached to this message.

Best regards,

**Comment by enisn** [comment]
Any reproduction steps?

Is that screenshot indicates your custom anchor image that covers your entire screen?

**Comment by GuyFlorent** [comment]
Yes, the component covers the entire screen and nothing else can be done. The scenario is simple: just place the component on a page and click the button to open it. You'll notice that it takes up the entire screen and nothing else is visible.

**Comment by GuyFlorent** [comment]
Hello, i am waiting for the answer please

**Comment by enisn** [comment]
Can you share the steps to reproduce? 
Example code?
